### PR TITLE
Remove mutex form ReadAheadmanager

### PR DESCRIPTION
**Pull request by daschuer**
because it is now only called from the Engine

### CODE REVIEW

**Comment by ywwg** [comment]
remove QMutex include

Add a command above the class declaration saying that the class is not threadsafe and can only be accessed from the engine thread.

**Comment by daschuer** [comment]
ready.

**Comment by kain88-de** [comment]
Can you also remove the `QMutex` includes to remove everything mutex related.

**Comment by daschuer** [comment]
Done!

File: src/engine/readaheadmanager.cpp [hunk 1]
```diff
@@ -133,8 +131,8 @@ void ReadAheadManager::addRateControl(RateControl* pRateControl) {
     m_pRateControl = pRateControl;
 }
 
+// Not threadsave, call from engine therad only
```
Comment (kain88-de) [inline src/engine/readaheadmanager.cpp]: I'm nitpicking here but it is 'thread'

**Comment by kain88-de** [comment]
LGTM besides the small comment

**Comment by daschuer** [comment]
Thank you for review.

**Pull request merged**